### PR TITLE
HTML: Remove all instances of `meta` element with attribute `http-equiv` of value `X-UA-Compatible` #7903 

### LINKED ISSUE #7903: HTML: Remove all instances of `meta` element with attribute `http-equiv` of value `X-UA-Compatible`
An example of a `meta` element to be removed would be `<meta http-equiv="X-UA-Compatible" content="IE=8">`

We do not need or want to render anything in an IE's legacy document mode.

Additional resources: [When to use legacy document modes]( https://msdn.microsoft.com/en-us/library/jj676917(v=vs.85).aspx)

### PR DESCRIPTION
**Pull request by shaharyar-shamshi**
Fixes #7903  
**outline of the solution**

All instances of `meta` element with attribute `http-equiv` of value `X-UA-Compatible` with refeence to #7903  had been removed from 

-  **src/test/resources/pages/commonAssertRegexTestPage.html** 

- **src/main/webapp/test/mashup.jsp**

### CODE REVIEW

**Comment by whipermr5** [comment]
Hello @shaharyar-shamshi, please follow our naming conventions mentioned in the document linked by the bot above :)

**Comment by shaharyar-shamshi** [comment]
Sorry for this I will not commit this mistake again .

**Comment by whipermr5** [comment]
@shaharyar-shamshi That's fine :) Please edit the PR title and description.

**Comment by whipermr5** [comment]
@shaharyar-shamshi Please open up the document to check the exact naming conventions (the bot isn't descriptive enough).

**Comment by shaharyar-shamshi** [comment]
I am sorry sir ( @whipermr5  ) I am not able to understand can you please help you more I will be thankful to you

**Comment by whipermr5** [comment]
Refer to [this](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr).

**Comment by shaharyar-shamshi** [comment]
Now I think it is perfect Sorry for so much inconvenience @whipermr5

**Comment by shaharyar-shamshi** [comment]
Sir ( @whipermr5  ) Is this pull Request fine or some modification is still Require.
As I wish to complete this task and move Forward to solve Another Issue

**Comment by chowyb** [comment]
@shaharyar-shamshi the naming conventions are now fine, but you may wish to check again to see if you have introduced any unwanted changes in this pull request.

**Comment by shaharyar-shamshi** [comment]
@chowyb  sir Actually on my local machine I was working on the issue by creating new branch and made the pull request from local machine to update the repositary and then merge it with the master branch and then further push and made pull request to merge with the teammates because of this  commit number 
31629c6 ,
02dbea5
10598aa
83d1388
1ccd3d1
1923bc4
Was by default derive from the pull request made by updating my repository if u merge present pull request then only last two commit would make change to teammates original code base  . If you wish then I can make another pull request

**Comment by chowyb** [comment]
@shaharyar-shamshi there should not be a need to create a new PR :)

By "unwanted changes" I was not referring to having extraneous commits, but rather, to check for whether the changes to the files in the current changeset (viewable by clicking on "Files changed" at the top of this page) contains the changes which you would like to push, and if there are any changes/files reflected there which you do not intend to push.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
         <div id="frameTopWrapper">
             <div id="logo">
                 <a href="/index.jsp">
-                <img width="150px" height="47px" src="/images/teammateslogo.jpg" alt="TEAMMATES" />
+                s<img width="150px" height="47px" src="/images/teammateslogo.jpg" alt="TEAMMATES" />
```
Comment (shaharyar-shamshi) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: this line is not intended to be committed   **sTEAMMATES**  **s** before image tag was by mistake ignore this whie merging

Comment (wkurniawan07) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: @shaharyar-shamshi, "ignore this while merging" is not an acceptable practice here. In fact it is explicitly stated in one of the items in the [PR template](https://github.com/TEAMMATES/teammates/blob/master/.github/PULL_REQUEST_TEMPLATE.md) that "[You should have] Gone through all the changes in this PR and ensured that no unintended changes were made".

If this kind of change is accidentally/otherwise introduced here, you need to revert it.

**Comment by shaharyar-shamshi** [comment]
@chowyb sir I have Reviewed the commit 
In commit number 
af1912d there was one commit by mistake

File: src/test/resources/pages/commonAssertRegexTestPage.html (hunk 1 above)
Comment (shaharyar-shamshi) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: okk sir this will be not be done again

Comment (shaharyar-shamshi) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: @wkurniawan07  sir should I make another pull request of this file . Or  Anything you would Suggest And Sorry for the mistake

Comment (wkurniawan07) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: Don't worry about all the mistakes, they can always be corrected. As @chowyb has said before, there is no need to open another PR; just push more changes here in this PR.

Comment (shaharyar-shamshi) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: Okk sir  thanks sir 😅😅 this is the learning process for me and thanks for your support

**Comment by wkurniawan07** [comment]
@shaharyar-shamshi you should check the "Files changed" tab again to ascertain that it really is the case.

**Comment by shaharyar-shamshi** [comment]
@wkurniawan07 sir I checked again there is no such unintended change now finally

**Comment by wkurniawan07** [comment]
@shaharyar-shamshi how about [this file](https://github.com/TEAMMATES/teammates/pull/8095/files#diff-95081120a29cba2262e96a9dc6ba9562)?

File: src/main/webapp/test/mashup.jsp [hunk 2]
```diff
@@ -9,7 +9,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="X-UA-Compatible" content="IE=8">
+
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Can completely remove the blank line instead.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml"><head>
 <head>
     <link href="/favicon.png" rel="shortcut icon" />
-    <meta content="IE=8" http-equiv="X-UA-Compatible" />
+
```
Comment (whipermr5) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: Same here.

File: src/main/webapp/test/mashup.jsp [hunk 4]
```diff
@@ -8,8 +8,7 @@
 
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta http-equiv="X-UA-Compatible" content="IE=8">
+  <head>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Delete the trailing spaces here.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 5]
```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml"><head>
 <head>
-    <link href="/favicon.png" rel="shortcut icon" />
-    <meta content="IE=8" http-equiv="X-UA-Compatible" />
+    <link href="/favicon.png" rel="shortcut icon" />
```
Comment (whipermr5) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: Delete the trailing space here.

**Comment by wkurniawan07** [comment]
For some reason Travis CI did not run here. I'll try to close and re-open the PR to see if it works.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 6]
```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml"><head>
+<html xmlns="http://www.w3.org/1999/xhtml">
+
```
Comment (whipermr5) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: Unrelated change.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 7]
```diff
@@ -1,13 +1,14 @@
...
+   
 <head>
     <link href="/favicon.png" rel="shortcut icon" /> 
     <title>TEAMMATES - Instructor</title>
     <link type="text/css" href="/stylesheets/main.css" rel="stylesheet" />
     <link type="text/css" href="/stylesheets/evaluation.css" rel="stylesheet" />
 
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-        <script src="/js/helper.js" language="JavaScript"></script>
+    <script src="/js/helper.js" language="JavaScript"></script>
```
Comment (whipermr5) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: Unrelated change.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 8]
```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml"><head>
+<html xmlns="http://www.w3.org/1999/xhtml">
```
Comment (whipermr5) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: Unrelated change.

File: src/test/resources/pages/commonAssertRegexTestPage.html [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
+   
 <head>
-    <link href="/favicon.png" rel="shortcut icon" />
-    <meta content="IE=8" http-equiv="X-UA-Compatible" />
+    <link href="/favicon.png" rel="shortcut icon" />
```
Comment (whipermr5) [inline src/test/resources/pages/commonAssertRegexTestPage.html]: There's still a trailing space at the end of the line.

File: src/main/webapp/test/mashup.jsp (hunk 4 above)
Comment (whipermr5) [reply src/main/webapp/test/mashup.jsp]: There are still two trailing spaces after `<head>`.

File: src/test/resources/pages/commonAssertRegexTestPage.html (hunk 7 above)
Comment (shaharyar-shamshi) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: @whipermr5   sir Actually I am going to attend the lecture can I perform the task after few hour .pardon me for delay .

Comment (whipermr5) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: Sure. Actually you might want to open a new PR and close this one when you continue. @wkurniawan07 discovered that the CI build somehow cannot begin on this PR.

Comment (shaharyar-shamshi) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: Sure sir @whipermr5  can trailing white space can't be removed directly using edit option on github

Comment (whipermr5) [reply src/test/resources/pages/commonAssertRegexTestPage.html]: The GitHub editor can do it; just delete the spaces at the end of the line:
screen shot 2017-10-23 at 12 03 57 pm
